kitchen.recipes.yml

```diff
@@ -0,0 +1,38 @@
+# Allows to test single recipes
+#
+# Must be run as local file (override) of kitchen.docker.yml
+# export KITCHEN_YAML=kitchen.docker.yml
+# export KITCHEN_LOCAL_YAML=kitchen.recipes.yml
+# See: https://kitchen.ci/docs/reference/configuration
+---
+verifier:
+  name: inspec
+  inspec_tests:
+    # recipe tests will use controls from these directories
+    - test/recipes
+    - test/resources
```

Comment:
can we remove test/resources from here?

Reply:
It is intentional: we might want to reuse some of the resource tests for the recipes which include those resources.